packages/psss/src/routes/PageRoutes.js

```diff
@@ -46,14 +52,11 @@ export const PageRoutesComponent = React.memo(({ canViewMultipleCountries, canVi
 ));
 
 PageRoutesComponent.propTypes = {
-  canViewMultipleCountries: PropTypes.func.isRequired,
-  canViewCountry: PropTypes.func.isRequired,
+  user: PropTypes.object.isRequired,
 };
 
 const mapStateToProps = state => ({
-  canViewMultipleCountries: () => checkIsMultiCountryUser(state),
```

Comment:
passing a function to `mapStateToProps`causes redux to re-render the component every time the state in the store changes anywhere. I refactored the component to only use the state it needs.

Reply:
That's very interesting, I didn't know it! Good on you for picking it up!